### DISCUSSION TITLE
Im new

### DISCUSSION THREAD

**Discussion by Praneeth8888** [Pallavi1926/Gym_Website__Using_HTML_CSS_JAVASCRIPT#196] [Q&A] [answered]
hey , im new to open source, i want contribute to this project , how can i ?

**Comment by rahulkumarparida** [comment, marked as answer]
if you are through Girlscript summer of code  you can read the how it works  section of the website [https://gssoc.girlscript.tech/info](url)
even then if you don't get it . It is as simple like you will fork and  download the repository of the given project and make the changes of the given issues they are facing  and upload it with a pull request the developer will review your changes and merge it  if he/she finds it useful
You should watch a youtube tutorial in my opinion
